src/components/map/components/external-realtime-map/ExternalMapButton.tsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import {Button} from '@atb/components/button';
+import {StyleSheet} from '@atb/theme';
+import {Map} from '@atb/assets/svg/mono-icons/map';
+
+type ExternalMapButtonProps = {
+  onPress: () => void;
+};
+export const ExternalMapButton = ({onPress}: ExternalMapButtonProps) => {
```

Comment:
Naming might become confusing in the future: `ExternalMapButton` vs `ExternalRealtimeMap` in the Map.tsx file. We could align this to the Firestore Config and roll `External Realtime Map` everywhere (e..g `ExternalRealtimeMapButton` and `external-realtime-map`)